doc/lsst.pipe.base/command-line-task-data-repo-howto.rst

```diff
@@ -0,0 +1,136 @@
+.. _command-line-task-data-repo-howto:
+
+#################################################################
+Using Butler data repositories and reruns with command-line tasks
+#################################################################
+
+:ref:`Command-line tasks <using-command-line-tasks>` use Butler data repositories for reading and writing data.
+Butler data repositories are specially-configured directories on a filesystem.
```

Comment:
Except that the statement is wrong.  A repository _is not_, in general, a directory.  A repository may be an object store, a database, or some other place where datasets can be stored.  Ideally, no one should ever have to know how to "anoint" a directory as a repository; that is a Butler implementation detail.